src/components/ContactListPage.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { filterList } from '../utils/index';
+
+import SearchBar from './SearchBar';
+import ContactList from './ContactList';
+
+const propTypes = {
+  searchValue: PropTypes.string.isRequired,
+  contactList: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.shape({
```

Comment:
I got some errors in console about prop types. This cuz you use `objectOf(shape(....))`. objectOf should be removed cuz shape it is an object

Reply:
Removed. Thanks!